src/table/virtualized-table/utils/get-height.ts

```diff
@@ -1,11 +1,11 @@
 import { COMMON_CELL_STYLING } from '../../styling-defaults';
 import { GeneratedStyling } from '../../types';
 import { fontSizeToRowHeight } from '../../utils/styling-utils';
-import { MIN_BODY_ROW_HEIGHT, MIN_HEADER_HEIGHT } from '../constants';
+import { MIN_BODY_ROW_HEIGHT, MIN_HEADER_HEIGHT, PADDING_TOP_BOTTOM } from '../constants';
 import { BodyStyle, Rect, Totals } from '../types';
 
 const getHeaderRowHeight = ({ fontSize = COMMON_CELL_STYLING.fontSize }: GeneratedStyling) =>
-  Math.max(MIN_HEADER_HEIGHT, fontSizeToRowHeight(fontSize));
+  Math.max(MIN_HEADER_HEIGHT, fontSizeToRowHeight(fontSize) + PADDING_TOP_BOTTOM) * 2;
```

Comment:
Why `* 2`? Looks like the header height in the rendering tests are too large?

Reply:
it should be ontSizeToRowHeight(fontSize) + PADDING_TOP_BOTTOM * 2) 🙃, already pushed it